prometheus_node_exporter.yaml

```diff
@@ -0,0 +1,21 @@
+name: prometheus-node-exporter
+version: 0.16
```

Comment:
Let's use `version-script: cat VERSION`, so that we automatically pick up the correct version from the repo.